client/post-editor/editor-revisions-list/style.scss

```diff
@@ -60,17 +60,79 @@
 	}
 }
 
+// Revision Navigation Buttons
+.editor-revisions-list__navigation {
+	display: flex;
+	flex-direction: row-reverse;
+	height: 40px;
+	color: $gray-text-min;
+	border-bottom: 1px solid darken($sidebar-bg-color, 5%);
+	.button:first-child,
+	.button:last-child {
+		border-radius: 0;
+	}
+
+	@include breakpoint( '>660px' ) {
+		flex-direction: row;
+		height: 24px;
+	}
+}
+
+.editor-revisions-list__prev-button,
+.editor-revisions-list__next-button {
+	display: inline-block;
+	width: 50%;
+	height: 40px;
+	vertical-align: middle;
+	background: transparent;
+	border: none;
+	&.button:disabled,
+	&.button[disabled],
+	&.button.disabled {
+		background: transparent;
+		color: lighten($gray-text-min, 30%);
+		border-color: darken($sidebar-bg-color, 5%);
+	}
+	.gridicon {
+		transform: rotate(-90deg);
+	}
+
```

Comment:
Try a `:focus { outline: none; }` to these to prevent this:

<img width="280" alt="screen shot 2018-01-28 at 10 28 41 am" src="https://user-images.githubusercontent.com/1587282/35483933-0f877ebe-0416-11e8-85fd-c51c79eececb.png">

Reply:
Thanks! I'll have a look at that.

Reply:
I've committed this change in 82353042640a344558c551ba3aa2411a25fa0ab3.